### PR TITLE
feat: fixed and upgraded example projects

### PR DESCRIPTION
**Pull request by reime005**
- Upgrade to react-native 0.59.3
- Replaced deprecated Slider component with @react-native-community/slider
- Linked JavaScriptCore.framework in iOS basic example project
- added android:usesCleartextTraffic="true" to fix bundle loading issue on Android

### CODE REVIEW

File: examples/basic/android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -14,6 +14,7 @@
 
     <application
       android:name=".MainApplication"
+      android:usesCleartextTraffic="true"
```
Comment (sibelius) [inline examples/basic/android/app/src/main/AndroidManifest.xml]: Is there any docs about this?

Comment (reime005) [reply examples/basic/android/app/src/main/AndroidManifest.xml]: https://github.com/facebook/react-native/issues/22375

**Comment by sibelius** [comment]
is this still needed after https://github.com/react-native-community/react-native-camera/pull/2193?

can you please rebase?

**Comment by simistern** [comment]
Could this be closed since its out of date?

**Comment by sibelius** [comment]
does we have RN60 example?

**Comment by sibelius** [comment]
closing, open a new PR with latest version with v3 please

**Pull request closed without merging**